src/ParseServer.js

```diff
@@ -468,9 +468,12 @@ function injectDefaults(options: ParseServerOptions) {
     }
   });
 
-  options.masterKeyIps = Array.from(
-    new Set(options.masterKeyIps.concat(defaults.masterKeyIps, options.masterKeyIps))
-  );
+  options.masterKeyIps =
+    options.masterKeyIps?.length > 0
+      ? Array.from(
+        new Set(options.masterKeyIps.concat(defaults.masterKeyIps, options.masterKeyIps))
+      )
+      : [];
```

Comment:
```suggestion
```

I think these lines can be removed completely

Reply:
yeah looking closer, fully agreed 😄

I was actually confused initially and didn't want to make some shit, especially because I'm not being able to run tests locally due to this error. any ideas?

```
~/dev/parse-server fix/masterKeyIps
❯ npm test

> parse-server@6.0.0-alpha.12 pretest /Users/stewan/dev/parse-server
> cross-env MONGODB_VERSION=${MONGODB_VERSION:=5.3.2} MONGODB_TOPOLOGY=${MONGODB_TOPOLOGY:=standalone} mongodb-runner start

  ◝ Starting a MongoDB deployment to test against...Error: Could not find download URL for version 5.3.2 {
  version: '5.3.2',
  arch: [ 'arm64', 'aarch64' ],
  platform: 'darwin',
  target: [
    { value: 'osx', priority: 1 },
    { value: 'osx-ssl', priority: 10 },
    { value: 'darwin', priority: 1 },
    { value: 'macos', priority: 1 }
  ],
  enterprise: false,
  cryptd: false
}
    at resolve (/Users/stewan/dev/parse-server/node_modules/mongodb-download-url/lib/index.js:177:15)
    at async getDownloadURL (/Users/stewan/dev/parse-server/node_modules/mongodb-download-url/lib/index.js:242:12)
npm ERR! Test failed.  See above for more details.
```